docker/case-handler.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+
+parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(description = "Helper script to add or remove cases from database")
+group  = parser.add_mutually_exclusive_group(required=True)
+group.add_argument('--add',         action='store_true',   help ="Add samples to db")
+group.add_argument('--remove',      action='store_true',   help ="Removes samples from db")
+parser.add_argument("--sampleid",   metavar = "SAMPLEID",  type = str,   required = False,    help = "Sample ID for sample to remove (defined under 'family' in yaml file")
+parser.add_argument("--instid",     metavar = "INSTID",    type = str,   required = False,    help = "Institute ID for sample to remove")
+parser.add_argument("--folder",     metavar = "FOLDER",    type = str,   required = False,    help = "Folder where the sample that should be imported is stored")
```

Comment:
so the --folder option decides what dir on the host that should be bound to /sample in the container, and --yaml is the path to the .yaml file relative to /sample. 

Here the tricky part is that some of the paths (e.g. the bam files) specified in the yaml-file should be visible to the container where the scout server is run. Also all paths given in the yaml-file must be relative to the /sample dir in the container. right?

Reply:
Yes for the first part. The comment regarding the bamfile path is exactly the problem I have also. There is something with the path of the bam files that doesn't work in the container. It can see the path to the vcf file, but it cannot find the bam files. I haven't yet had time to look more into this and how to fix it.